gammapy/maps/axes.py

```diff
@@ -0,0 +1,145 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+
+import numpy as np
+from astropy.time import Time
+
+
+class TimeAxis:
+    """Class representing a time axis.
+
+    Provides methods for transforming to/from axis and pixel coordinates.
+    A time axis can represent non-contiguous sequences of time intervals.
+
+    Parameters
+    ----------
+    time_min : `~astropy.time.Time`
+        Array of edge time values.
+    time_max : `~astropy.time.Time`
+        Array of edge time values.
+    interp : str
+        Interpolation method used to transform between axis and pixel
+        coordinates.  Valid options are 'log', 'lin', and 'sqrt'.
+    name : str
+        Axis name
+    """
+    node_type = "edges"
+    def __init__(self, time_min, time_max, name="", interp="lin",):
```

Comment:
I think we could safely assume the default name as `"time"`, or even hard-code it to `"time"` (not sure about the latter, but I think there will never be two time axes in the same object...)